source/assets/js/playground/utils.ts

```diff
@@ -66,3 +68,44 @@ export function errorToDiagnostic(error: Exception | unknown): Diagnostic {
     };
   }
 }
+
+export function debugToDiagnostic(logItem: ConsoleLogDebug): Diagnostic {
+  return {
+    from: logItem.options.span.start.offset,
+    to: logItem.options.span.end.offset,
+    severity: 'info',
+    message: logItem.message,
+  };
+}
+
+export function warnToDiagnostic(
+  logItem: ConsoleLogWarning
+): Diagnostic | null {
+  if (!logItem.options.span) return null;
+  return {
+    from: logItem.options.span.start.offset,
+    to: logItem.options.span.end.offset,
+    severity: 'warning',
+    message: logItem.message,
+  };
+}
+
+export function logsToDiagnostics(logs: ConsoleLog[]): Diagnostic[] {
+  const diagnostics = logs.flatMap(log => {
+    if (log.type === 'error') return errorToDiagnostic(log.error);
+    else if (log.type === 'warn') return warnToDiagnostic(log);
+    else if (log.type === 'debug') return debugToDiagnostic(log);
+    else return null;
+  });
+  // Remove empties
+  return diagnostics.filter(
+    (diagnostic): diagnostic is Diagnostic => !!diagnostic
+  );
+}
+
+export const customLoader: Importer<'sync'> = {
+  canonicalize() {
+    throw new Error(PLAYGROUND_LOAD_ERROR_MESSAGE, {cause: 'Test'});
```

Comment:
Is this `cause` supposed to be here? What is it doing?

Reply:
@nex3 It's not needed, I forgot to remove it. Removed in https://github.com/sass/sass-site/pull/754/commits/e81e4a0ec6288e67e79e738769d7d555342a69d2